aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -251,7 +251,7 @@ Tensor empty_like(
   Tensor result;
 
   auto memory_format =
-      optional_memory_format.value_or(MemoryFormat::Contiguous);
+      optional_memory_format.value_or(MemoryFormat::Preserve);
```

Comment:
are you going to change new_empty (below)?  I don't know what the right answer is, just asking.

Reply:
Eventually yes, as we start increasing support.